nf_core/bump_version.py

```diff
@@ -83,11 +83,6 @@ def bump_nextflow_version(lint_obj, new_version):
     nfconfig_newstr = "nextflowVersion = '>={}'".format(new_version)
     update_file_version("nextflow.config", lint_obj, nfconfig_pattern, nfconfig_newstr)
 
-    # Update travis config
-    nfconfig_pattern = r"NXF_VER=[\'\"]?{}[\'\"]?".format(current_version.replace('.',r'\.'))
```

Comment:
This code needs to be replicated for the new GitHub Actions scripts. Needs to change these lines:

https://github.com/nf-core/tools/blob/25335b387589008838c83fb0a73808cc0f9a7cbb/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/.github/workflows/ci.yml#L14-L15

Reply:
Yep. Realised that already looking through the changes and pushed a commit Im going to test locally.

Reply:
See https://github.com/nf-core/tools/pull/502/commits/7f1f2398a1a64fe8ba63825535ad3f9f7b2d3361